### PR TITLE
[docker] fixes #10 Dockerfile for Skycoin .NET app developers

### LINKED ISSUE #10: [docker] Virtualize workspaces for developers of Skycoin .NET apps
Image name `simelotech/skycoindev-dotnet` and a single `develop` tag.

- [ ] .NET CLI tools built `FROM skycoin/skycoindev-cli:develop` 
- [ ] .NET CLI tools and Docker built `FROM skycoin/skycoindev-cli:dind` 
- [ ] .NET tools and VS Code IDE built `FROM skycoin/skycoindev-vscode:develop` 
- [ ] .NET tools, Docker and VS Code IDE built `FROM skycoin/skycoindev-vscode:dind`

### PR DESCRIPTION
**Pull request by stdevHan**
Fixes #10 

Changes:
- Create file dockefile by container

### CODE REVIEW

File: docker/images/dev-cli/README.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
+# Supported tags and respective `Dockerfile` links
+
+## Simple Tags
+
+-	[`develop` (*docker/images/dev-cli/Dockerfile*)](https://github.com/simelo/libskycoin-dotnet/blob/develop/docker/images/dev-cli/Dockerfile)
+
+# Libskycoin DotNet CLI development image
```
Comment (olemis) [inline docker/images/dev-cli/README.md]: DotNet => .NET

File: docker/images/dev-cli/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,19 @@
+# Creates an image for skycoin development
+FROM skycoin/skycoindev-cli:develop
+
+ENV MONO_VERSION 4.8.0.524
+
+RUN apt-get update \
+  && apt-get install -y curl \
+  && rm -rf /var/lib/apt/lists/*
+
+RUN apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys 3FA7E0328081BFF6A14DA29AA6A19B38D3D831EF
```
Comment (olemis) [inline docker/images/dev-cli/Dockerfile]: Further keys needed for stable builds , ask @stdevNorge

File: docker/images/dev-cli/README.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+to ease go development and version control with git.
+
+# How to use this image
+
+## Initialize your development environment.
+
+```sh
+$ mkdir src
+$ docker run --rm \
+    -v ${PWD}/src:/usr/local/src skycoin/skycoindev-dotnet:develop \
```
Comment (olemis) [inline docker/images/dev-cli/README.md]: What are these volumes for ?

**Comment by olemis** [comment]
@stdevHan @hanibal92 pls apply changes suggested this PR 👆 to see if build trigger works in Docker Cloud

File: docker/images/dev-cli/Dockerfile (hunk 2 above)
Comment (stdevHan) [reply docker/images/dev-cli/Dockerfile]: the one that suggests those of mono

**Comment by olemis** [comment]
needs a different approach

**Pull request closed without merging**